src/esm/speech.js

```diff
@@ -238,12 +238,13 @@ class StreamingSynthesisConnection {
 };
 
 export class Speech {
-  constructor(apiKey) {
+  constructor(apiKey, baseUrl = null) {
```

Comment:
Just because it's altering a public facing method/fn, worth adding a test to validate that passing a value here results in use of the different base for an API call.